recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   number: 0
-  skip: true          # [win and not (py34 or py35)]
+  skip: true          # [win and py27]
```

Comment:
It doesn't. Since the VC version of py35 and py36 are the same, there's no need to build twice.